xml/cap_depl_eks.xml

```diff
@@ -513,4 +513,225 @@ secrets:
+   </para>
+   <itemizedlist>
+    <listitem>
+     <para>
+      &cap; has been successfully deployed on Amazon EKS as described earlier in this chapter (see <xref linkend="cha.cap.eks"/>)
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      The <link xlink:href="https://aws.amazon.com/cli/">AWS Command Line Interface (CLI)</link>
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      The <link xlink:href="https://www.openssl.org/docs/man1.0.2/apps/openssl.html">OpenSSL command line tool</link>
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      Ensure the user or role running the broker has an IAM policy set as specified <link xlink:href="https://github.com/awslabs/aws-servicebroker/blob/master/docs/install_prereqs.md#iam">here</link>
+     </para>
+    </listitem>
+   </itemizedlist>
+  </sect2>
+  <sect2 xml:id="sec.cap.aws-service-broker-setup">
+   <title>Setup</title>
+   <procedure>
+    <step>
+     <para>
+      Create the required DynamoDB table where the AWS service broker will store its data. This example creates a table named <literal>awssb</literal>:
+     </para>
+<screen>&prompt.user;aws dynamodb create-table \
+		--attribute-definitions \
+			AttributeName=id,AttributeType=S \
+			AttributeName=userid,AttributeType=S \
+			AttributeName=type,AttributeType=S \
+		--key-schema \
+			AttributeName=id,KeyType=HASH \
+			AttributeName=userid,KeyType=RANGE \
+		--global-secondary-indexes \
+			'IndexName=type-userid-index,KeySchema=[{AttributeName=type,KeyType=HASH},{AttributeName=userid,KeyType=RANGE}],Projection={ProjectionType=INCLUDE,NonKeyAttributes=[id,userid,type,locked]},ProvisionedThroughput={ReadCapacityUnits=5,WriteCapacityUnits=5}' \
+		--provisioned-throughput \
+			ReadCapacityUnits=5,WriteCapacityUnits=5 \
+		--region ${AWS_REGION} --table-name <replaceable>awssb</replaceable>
+</screen>
+    </step>
+    <step>
+     <para>
+      Wait for the table to finish creating. The <literal>TableStatus</literal> should be <literal>ACTIVE</literal> when it's ready. Check the status using the <command><link xlink:href="https://docs.aws.amazon.com/cli/latest/reference/dynamodb/describe-table.html">describe-table</link></command> command: 
+     </para>
+     <screen>aws dynamodb describe-table --table-name <replaceable>awssb</replaceable></screen>
+    </step>
+    <step>
+     <para>
+      Set a name for the &kube; namespace you will install the service broker to. This name will also be used in the service broker URL:
+     </para>
+     <screen>&prompt.user;BROKER_NAMESPACE=aws-sb</screen>
+    </step>
+    <step>
+     <para>
+      Create a server certificate for the service broker:
+     </para>
+     <substeps>
+      <step>
+       <para>
+        Get the CA certificate:
+       </para>
+       <screen>kubectl get secret --namespace scf --output jsonpath='{.items[*].data.internal-ca-cert}' | base64 -di > ca.pem</screen>
```

Comment:
Maybe suggest to do this in a separate directory, where it won't conflict with other ca.* files?

Reply:
Added in https://github.com/SUSE/doc-cap/pull/289/commits/dca9e2098eb6f86bdb52bbc14c588667bc8163aa

Reply:
awesome sauces!